.github/workflows/npm-publish.yml

```diff
@@ -6,6 +6,19 @@ on:
       - master
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 19
+          registry-url: https://registry.npmjs.org/
+      - run: yarn install
+      - run: yarn build
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Is it included into the `publish-and-notify`

Reply:
yes, it is indeed included in publish.

Reply:
so you can remove it from the PR